src/utils/misc.ml

```diff
@@ -120,9 +120,7 @@ let may_map f x = Option.map ~f x
 (* File functions *)
 
 let remove_file filename =
-  try
-    if Sys.is_regular_file filename
-    then Sys.remove filename
```

Comment:
Because my local version of the compiler didn't recognize the `Sys.is_regular_file`, but now I see a 5.1 switch does. I'm unsure if CI is on a sufficiently high version of OCaml to compile this &mdash; I'll find out when CI passes (or doesn't).

Reply:
It does. I reverted the diff you commented on.